assets/js/modules/analytics/dashboard/dashboard-widget.js

```diff
@@ -274,4 +281,4 @@ class AnalyticsDashboardWidget extends Component {
 	}
 }
 
-export default AnalyticsDashboardWidget;
+export default withSelect( ( select ) => ( { dateRange: select( STORE_NAME ).getDateRange() } ) )( AnalyticsDashboardWidget );
```

Comment:
This is a bit of a nitpick, but this would be a little cleaner over multiple lines.

Reply:
Updated.